spec/controllers/challenge_assignments_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe ChallengeAssignmentsController do
+  include LoginMacros
+  include RedirectExpectationHelper
+  
+  before(:each) do
```

Comment:
This shouldn't need a before block -- just `let(:collection) { create(:collection) }` and it will be executed when referenced. (In fact, if I recall correctly, `let` doesn't work in before blocks.)